components/loadercontainer/overview.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Overview
+page_title: LoaderContainer Overview
+description: Overview of the LoaderContainer for Blazor.
+slug: loadercontainer-overview
+tags: telerik,blazor,loader,container,loadercontainer,overview
+published: True
+position: 0
+---
+
+# LoaderContainer Overview
+
+This article provides information about the LoaderContainer for Blazor and its core features.
+
+The LoaderContainer provides an animated indicator, a panel, and an overlay that can be used when the application is performing a time-consuming operation, for example, loading data.
+
+## Basic LoaderContainer
+
+To add a Telerik LoaderContainer to your Blazor application, use the `<TelerikLoaderContainer>` tag and show it when needed by your app by using its `Visible` paramter. You can also control its [visual appearance]({%slug loadercontainer-appearance%}) through parameters.
+
+![](images/loadercontainer-overview-example.gif)
+
+````CSHTML
+@*Show the LoaderContainer until the data for the Grid is loaded.*@
+
+<TelerikLoaderContainer Visible="@(GridData == null ? true: false)"></TelerikLoaderContainer>
```

Comment:
Consider moving this as a separate example in the Examples section (soething like Loading Panel or Grid during initial load, so we could link that from its own docs when we document the integration). Perhaps could benefit from a common container for the grid and the loader.

Consider making the basic example a div with a button in it and the button doing something long to show the loader panel